core/src/test/java/org/ehcache/EhcacheBasicCrudBase.java

```diff
@@ -730,6 +731,13 @@ public void describeTo(final Description description) {
      * Keys for which access results in a thrown {@code Exception}.  This set may be empty.
      */
     private final Set<String> failingKeys;
+    
+    /**
+     * Keys for which success should be reported in {@link BulkCacheLoadingException}
+     */
+    private Set<String> successKeys;
+    
+    private boolean isBulkCacheLoadingExceptionEnabled = false;
```

Comment:
Why do you need these two additional fields?
I was expecting you would be able to use the `org.ehcache.EhcacheBasicCrudBase.FakeCacheLoaderWriter#completeFailureKey` and `org.ehcache.EhcacheBasicCrudBase.FakeCacheLoaderWriter#failingKeys` as they are used in the `BulkCacheWritingException` testing